core/eolearn/core/eodata.py

```diff
@@ -255,6 +255,23 @@ def add_feature(self, feature_type, feature_name, value):
         """
         self._check_if_dict(feature_type)
         self[feature_type][feature_name] = value
+    
+    def rename_feature(self,feature_type,feature_name,new_feature_name):

```

Comment:
It might be better to go with the feature parser here, so that we enforce features to be of the same form, sort of like this:

```
def rename_feature(self, feature): 
    feature_type, feature_name, new_feature_name = self._parse_features(feature, new_names=True)
```
The docstring is also shorter then

Reply:
Additionally, you could add a check if the new feature name is the same as the old one and not waste CPU in this case.